README.md

```diff
@@ -1,167 +1,169 @@
-Total Users using the latest release
+# VTOL
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/45333346/155147598-71510387-7b83-4dea-a1b8-b322888d8a3e.png" />
+</p>
+
+Users on the latest release:
+
 ![GitHub release (latest by date)](https://img.shields.io/github/downloads/BigSpice/VTOL/latest/total?style=flat-square)
-<br /><br />
-<br /><br />
-**This application installs the [Titanfall 2 Northstar Client](https://northstar.tf),Installs Skins, and optionally installs compatible mods for it created by various modders.**
-<br /><br />
+
+
+## This is a launcher for the [Titanfall 2 Northstar Client](https://northstar.tf)
+
+
 ![VTOL_Showcase_Trim](https://user-images.githubusercontent.com/23240514/152448826-0a92b6d8-3cce-4107-911c-0a1480ad8a8e.gif)<br /><br /><br />
 
-Features:
-<br />
-• Updating [Northstar](https://northstar.tf) installations.
-<br />
-• Easy installation of the [Northstar Client](https://northstar.tf)
-<br />
-• Verification of a [Northstar](https://northstar.tf) installation (if you did it manually)
-<br />
-• Viewing installed mods
-<br />
-• Toggling installed [Northstar](https://northstar.tf) mods
-<br />
-• Downloading and installing [Northstar](https://northstar.tf) mods from a GitHub/GitLab repository
-<br />
-• Installing downloaded [Northstar](https://northstar.tf) mods (.zip files)
-<br />
-• Installing and injecting various types of Titanfall 2 player/weapon skins
-<br />
-• Booting up a dedicated server for the [Northstar Client](https://northstar.tf)
-<br />
-• Install mods featured in the [Thunderstore](https://northstar.thunderstore.io) collection.
-<br />
-• Boot Northstar
-<br /><br /><br />
-WIP Features:
-<br />
-• Create custom servers using this application to fine tune setups.
-<br />
-<br />
-• Manage dedicated [Northstar](https://northstar.tf) servers.
-<br /><br />
+## Features:
 
-_Suggestions/bug reports are highly appreciated. Please create a new [issue](https://github.com/BigSpice/VTOL/issues/new] and fill out the fields._
-<br />
-<br /><br />
+* Easilly install, update and launch [Northstar](https://northstar.tf)
+
+* Manage installed [Northstar](https://northstar.tf) mods
+
+* Downloading and installing [Northstar](https://northstar.tf) mods from a GitHub/GitLab repository
+
+* Installing downloaded [Northstar](https://northstar.tf) mods (.zip files)
+
+* Easilly isntall custom Wepaon/Pilot Skins
+
+* Easilly start a [Northstar](https://northstar.tf) Dedicated Server
+
+* [Thunderstore](https://northstar.thunderstore.io) Mod Browser
+
+
+### WIP Features:
+
+* Create custom servers using this application to fine tune setups.
+
+* Manage dedicated [Northstar](https://northstar.tf) servers.
+
+### Suggestions/Bug reports?
```

Comment:
preferably keep this at `_Suggestions/bug reports are highly appreciated. Please create a new [issue](https://github.com/BigSpice/VTOL/issues/new] and fill out the fields._`
(remove the header)

Reply:
I'll change the text back, but I don't see the point in removing the header, why do you think it should be removed?

Also, there was a missing bracket on the link

Reply:
i see, you could change the header to "Contribute" or something

Reply:
alright, updated the header